io.github.tdesktop64.TDesktop.yml

```
@@ -0,0 +1,554 @@
+id: io.github.tdesktop64.TDesktop
```

```suggestion
id: io.github.tdesktop_x64.TDesktop
```

This would be the correct app-id